api/src/templates/new_member.body.html

```diff
@@ -1,167 +1,194 @@
+<h3>About Stockholm Makerspace:</h3>
+<p>Stockholm Makerspace is a non-profit organization “for members, by members”. This means that you too, as a member, are encouraged to participate and collaborate with the development of the organization.</p>
 
-<h3>I would like to have certain tool/thing. Can I buy that thing?</h3>
-<p>Purchases for the organization should be democratic as much as possible. Unfortunately, we only have 300m2 of space and it is packed with things, so donations and purchases must be approved by the board before purchasing and transport to the premises. If you want to get compensation for expenses that are made for our common machine park then you need to have the purchase approved by a board member or room manager before purchasing. Save the purchase receipt so that our treasurer can handle the accounting. In the case of more expensive purchases, the Board needs to meet to make a joint decision before a purchase can be made. You can also write down what you want on Slack or in the Facebook-group.</p>
-
-<h3>Can I make a prototype at the Stockholm Makerspace?</h3>
-<p>Yes, if there is room and if you do not sell the prototype. You also should not have formed a company yet. Stockholm Makerspace is a non-profit association. In order to be able to continue like this, we must not conduct commercial activities. This means that you are not allowed to make products for sale.</p>
-
-<h3>Is it true I can keep my own storage box here?</h3>
-<p>Yes! Purchase your own storage box of size 50x39x26 cm (32L). Label it with a label from <a href="https://wiki.makerspace.se/Memberbooth">Memberbooth</a> and find a place for it on the shelves in the corridor. Boxes can be purchased at places like <a href="https://www.clasohlson.com/se/F%C3%B6rvaringsl%C3%A5da-SmartStore-Home/34-1552-31">Clas Ohlson</a>.</p>
+<h3>Getting Started:</h3>
+<p>First of all, make sure you have all you need to get started:</p>
+<ul>
+    <li>Make sure you have installed the Accessy app. If you don´t, you will find all the information you need on the <a href="https://wiki.makerspace.se/Accessy">Accessy wiki page</a>.</li>
+    <li>Make sure to download Slack (from <a herf="https://play.google.com/store/apps/details?id=com.Slack&hl=en_US&pli=1">Google Play</a> or <a href="https://apps.apple.com/se/app/slack/id618783545">Apple Store</a>), since it is the main means of communication at the makerspace. If you need help, check the <a href="https://wiki.makerspace.se/Slack">Slack Wiki page</a>.</li>
+    <li>Here are links to other important online resources</li>
+    <ul>
+        <li>&#128100; <a href="https://medlem.makerspace.se/">Din medlemssida</a></li>
+        <li>&#x1F6D2; <a href="https://medlem.makerspace.se/shop">Medlemsshop</a></li>
+        <li>&#128198; <a href="https://www.makerspace.se/kalendarium">Kalendarium</a></li>
+        <li>&#128213; <a href="https://wiki.makerspace.se/V%C3%A4lkommen_till_Stockholm_Makerspace">Wiki page</a></li>
```

Comment:
Not sure what emojis are used here, they don't render for me:

![image](https://github.com/makerspace/makeradmin/assets/7572427/089aaa01-668e-4b20-afbb-3b091a2fff64)

Perhaps this has something to do with the file encoding not being set properly?